cmd/context/delete.go

```diff
@@ -28,12 +29,18 @@ import (
 func DeleteCMD() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "delete",
-		Args:  utils.ExactArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
-		Short: "Delete a context",
+		Args:  utils.MinimumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
+		Short: "Delete one or more contexts",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			args[0] = okteto.AddSchema(args[0])
-			args[0] = strings.TrimSuffix(args[0], "/")
-			return Delete(args[0])
+			for _, arg := range args {
+				arg = okteto.AddSchema(arg)
+				arg = strings.TrimSuffix(arg, "/")
+				if err := Delete(arg); err != nil {
+					return err
+				}
+			}
+			analytics.TrackContextDelete(len(args))
```

Comment:
if I understood correctly @andreafalzetti, your proposal is that we could store if there is an error in the delete logic and track it along with the number of contexts to delete right? After that we could return the possible error to the user

Reply:
@AdrianPedriza I think we're good without it, we have no plans to act on that info anyway